sqlc/private/actions.bzl

```diff
@@ -116,7 +129,7 @@ def sqlc_compile(ctx, config_file, config_path_depth, srcs, out):
     ctx.actions.run_shell(
         tools = [toolchain.release.sqlc],
         # TODO(Windows) Figure out path handling for windows
-        command = "cd {} && {}/{} generate".format(
+        command = "cd {} && HOME=$(PWD) {}/{} generate".format(
```

Comment:
I think it does not really matter what HOME env var value is in this situation. Internally the HOME is used to create a `.cache` dir that is used by sqlc wasm plugin. I think we can put anything there just to satisfy sqlc